Data/Sys/GameSettings/RM8E01.ini

```diff
@@ -671,20 +528,52 @@ d200abd8 00000003
 c3ff0060 ec1f0032
 efe1002a 00000000
 e0000000 80008000
-*This cheat makes it so Rudder Madness is more balanced
+*Makes it so Rudder Madness is more balanced
+
+$Minigame - Surf's Way Up: All Targets Worth 100 Points [gamemasterplc]
+48000000 800030c8
+de000000 80008180
+3000b34c 881e0000
+1400b348 3ba00064
+e0000000 80008000
+*Makes it so Surfs Way Up is more balanced
+
+$Minigame Replacement - Bob-ombs Away ➜ Power Trip [gamemasterplc]
+282287CC 00000012
+022287CC 00000011
+E2000001 80008000
+
+$Minigame Replacement - Cardiators ➜ Frozen Assets [gamemasterplc]
+282287CC 00000039
+022287CC 00000031
+E2000001 80008000
+
+$Minigame Replacement - Cut from the Team ➜ Snipe for the Picking [gamemasterplc]
+282287CC 00000024
+022287CC 00000025
+E2000001 80008000
 
-$Minigame - Cosmic Slalom: Infinite Boosts
-48000000 800030C8
-DE000000 80008180
-30012D2C 2C000000
-14012D54 38030000
-E0000000 80008000
-*This cheat makes it so Cosmic Slalom is more balanced
+$Minigame Replacement - Glacial Meltdown ➜ Saucer Swarm [gamemasterplc]
+282287CC 00000027
+022287CC 00000026
+E2000001 80008000
+
+$Minigame Replacement - Gun the Runner ➜ Snow Way Out [gamemasterplc]
+282287cc 0000000f
+022287cc 00000015
+e2000001 80008000
+
+$Minigame Replacement - Specter Inspector ➜ Loco Motives [gamemasterplc]
+282287CC 00000030
+022287CC 0000002F
+E2000001 80008000
+
+$Minigame Replacement - Thrash 'n' Crash ➜ Swervin' Skies [gamemasterplc]
+282287CC 00000016
+022287CC 00000013
+E2000001 80008000
 
-$Minigame - Surf's Way Up: All Targets Worth 100 Points
-48000000 800030C8
-DE000000 80008180
-3000B34C 881E0000
-1400B348 3BA00064
-E0000000 80008000
-*This cheat makes it so Surfs Way Up is more balanced
+$Minigame Replacement - You're the Bob-omb ➜ Lava Lobbers [gamemasterplc]
+282287CC 00000037
+022287CC 0000002E
+E2000001 80008000
```

Comment:
newline

Reply:
Git was forcing to change to CRLF had to reverse it sorry for the trouble